Assignment2/manumazvi.ttl

```diff
@@ -0,0 +1,14 @@
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix : <http://example.org/elements>.
+
+:Class01 :includes :Sensor029; 
+		:includes :Computer101.
+
+:Sensor029 :hasMeasurement :Measurement8401.
+
+:Measurement8401 :hasTemperature "25"^^xsd:integer;
+             :atTime "2006-08-12"^^xsd:dateTime.

```

Comment:
This should be actually an xsd:date, since you have removed the time from the values. 
But I am accepting it.